basket/news/tests/test_tasks.py

```diff
@@ -771,3 +786,18 @@ def test_timestamps_nothin_cached(self, cache_mock, sfmc_mock):
             'FXA_ID': data['uid'],
             'NewEmailAddress': data['email'],
         })
+
+
+class GmttimeTests(TestCase):
+    @patch('basket.news.tasks.now')
+    def test_no_basetime_provided(self, now_mock):
+        # original time is 'Fri, 09 Sep 2016 13:33:55 GMT'
```

Comment:
I'm really confused by this test. If I use JS to create a date using the timestamp 1473428035498 (milliseconds), I get a date of `Fri, 09 Sep 2016 13:33:55 GMT`, yet the output in the test is 20:43 rather than 13:43.

Even more strange, if type gmttime's code into my console, the output is:
> 'Fri, 09 Sep 2016 12:43:55 GMT'

Reply:
@pmac - any idea why the test would return such different results to running the same code in the python module?

Reply:
I think it may be because I told you wrong... I think we should be using `datetime.datetime.fromtimestamp` instead of the `utcfromtimestamp` variety. In experimenting with it that seems to be the right thing. My apologies.